server/report/report-service.js

```diff
@@ -102,4 +104,32 @@ export default class {
       path: path.join(options.path, filename)
     });
   }
+  remapSourceMaps(options) {
+    const remapIstanbul = Npm.require('remap-istanbul');
+    const cwd = process.cwd();
+    process.chdir(Conf.COVERAGE_APP_FOLDER);
+
+    // Create output directory if not exists
+    let remapFolder = `${Conf.COVERAGE_EXPORT_FOLDER}-remap`;
```

Comment:
We could use a different filename for each report, but the report type `html` always generate the same files (`index.html`, and details page's `index.html` in their respective folder), which would be overwritten, so I need a different output folder to save the remapped `html` report.

Being so, I prefer to keep grouped the compiled JavaScript reports in one folder and the remapped original sources in other. We could create another folder inside output folder for the `html` report like `.coverage/html_remap` and keep all the remap files inside the output folder.

We could also create a `.coverage/remap` folder and save all the remapped reports inside it, which, IMO would be better because of grouping the reports based on the sources.

What do you think about this @serut?